rasa/utils/log_utils.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import sys
+from typing import Any, Dict
+
+import structlog
+from structlog_sentry import SentryProcessor
+from rasa.plugin import plugin_manager
+
+
+def _anonymizer(
+    _: structlog.BoundLogger, __: str, event_dict: Dict[str, Any]
+) -> Dict[str, Any]:
+    """Anonymizes event dict."""
+    anonymization_pipeline = plugin_manager().hook.get_anonymization_pipeline()
+
+    if anonymization_pipeline:
+        event_dict["event"] = anonymization_pipeline.log_run(event_dict["event"])
+    return event_dict
+
+
+def configure_logging() -> None:
```

Comment:
should we actually rename this `configure_structlog` instead? to differentiate it from the std library `logging`?